requirements/requirements.in

```diff
@@ -53,7 +53,7 @@ seaborn
 matplotlib
 statsmodels
 scikit-learn
-numpy
+numpy>=1.24.0
```

Comment:
1.19 was breaking in the CI req freeze. Keep in mind this is only used to generate the requirements for our own project, and doesn't impact what versions of numpy users can use.